### PR TITLE
Add buck support to map_from_entries Presto function

### PR DESCRIPTION
**Pull request by darrenfu**
Summary:
In previous PR: https://github.com/facebookincubator/velox/pull/3417 (D46169043), the buck TARGETS file changes are overlooked. Thus it causes many internal build failures.

This change is to amend those BUCK changes.

Differential Revision: D46700832

### CODE REVIEW

File: velox/functions/prestosql/MapFromEntries.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    auto& inputValueVector = inputArray->elements();
+    exec::LocalDecodedVector decodedValueVector(context);
+    decodedValueVector.get()->decode(*inputValueVector);
+    auto valueRowVector = decodedValueVector->base()->as<RowVector>();
+    auto keyValueVector = valueRowVector->childAt(0);
+
+    // Validate all map entries and map keys are not null.
+    if (decodedValueVector->mayHaveNulls() || keyValueVector->mayHaveNulls()) {
+      context.applyToSelectedNoThrow(rows, [&](vector_size_t row) {
+        auto size = inputArray->sizeAt(row);
```
Comment (xiaoxmeng) [inline velox/functions/prestosql/MapFromEntries.cpp:90]: nit: const auto size offset

**Review by xiaoxmeng — APPROVED** [review]
@darrenfu LGTM. Thanks!

File: velox/functions/prestosql/tests/MapFromEntriesTest.cpp [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  auto typedExpr =
+      makeTypedExpr("map_from_entries(c0)", asRowType(rowVector->type()));
+  std::vector<VectorPtr> results(1);
+
+  exec::ExprSet exprSet({typedExpr}, &execCtx_);
+  exec::EvalCtx evalCtx(&execCtx_, &exprSet, rowVector.get());
+  exprSet.eval(rows, evalCtx, results);
+
+  ASSERT_EQ(results[0]->size(), 2);
+}
```
Comment (xiaoxmeng) [inline velox/functions/prestosql/tests/MapFromEntriesTest.cpp:240]: Can you add test cases to cover the duplicate cases? Thanks!

Comment (xiaoxmeng) [reply velox/functions/prestosql/tests/MapFromEntriesTest.cpp:240]: There is already one. Thanks!
